src/index.css

```diff
@@ -102,4 +102,14 @@ code {
 
 .react-datepicker-wrapper {
   flex-grow: 1;
+}
+
+.addButton{
+  margin-top: 2em;
+  top: 50%;
+  right: 0px;
+}
+
+.tableButton{
+  float: right;
```

Comment:
This CSS should go in a `GeneralInformation.css` file since it is only valid for the GeneralInformation component.